src/pe/import.rs

```diff
@@ -1,13 +1,47 @@
 use alloc::borrow::Cow;
 use alloc::vec::Vec;
+use core::fmt::{LowerHex, Debug};
 
 use scroll::{self, Pread};
+use scroll::ctx::TryFromCtx;
 use error;
 
 use pe::section_table;
 use pe::utils;
 use pe::data_directories;
 
+
+pub const IMPORT_BY_ORDINAL_32: u32 = 0x8000_0000;
+pub const IMPORT_BY_ORDINAL_64: u64 = 0x8000_0000_0000_0000;
+pub const IMPORT_RVA_MASK_32: u32 = 0x8fff_ffff;
+pub const IMPORT_RVA_MASK_64: u64 = 0x0000_0000_8fff_ffff;
+pub const SIZEOF_IMPORT_ADDRESS_TABLE_ENTRY_32: usize = 4;
+pub const SIZEOF_IMPORT_ADDRESS_TABLE_ENTRY_64: usize = 8;
```

Comment:
Since this value is only used for the import lookup table (not the address table), I think it would be better to use 'lookup' in the name, even though both tables have the same entry size (and sometimes the same entry format).